src/poppler/document.py

```diff
@@ -70,42 +70,50 @@ def create_page(self, index):
         return Page(self._document.create_page(index))
 
     @property
+    @since(0, 46)
     @ensure_unlocked
     def author(self):
         return str(self._document.get_author())
```

Comment:
Just to be sure you understood my comment, here is an example:

```suggestion
    @property
    @ensure_unlocked
    def author(self):
        try:
            value = self._document.get_author()
        except AttributeError:
            value = self._document.info_key('author')
        return str(value)
```
And so on for other properties. This way, you should be able to re-enable some tests.

Reply:
Is there any reason not to always use `.info_key` all the time here?

Reply:
This is a good question. No, I don't think there is any reason, except that the function exists. We could definitively use info_key() and set_info_key() all the time.